### DISCUSSION TITLE
assets only named slugs?

### DISCUSSION THREAD

**Discussion by ZeFish** [ObsidianPublisher/obsidian-github-publisher#257] [Q&A — Plugin]
I don't know what I'm doing wrong here...
<img width="126" alt="Screenshot 2023-12-18 at 7 03 58 PM" src="https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/24323/b49d5c66-f122-4015-a53e-b5e4c60800a8">

**Comment by Mara-Li** [comment]
Closed because you don't respect the get help tutorial.
Feel free to reopen an question with mandatory information.
